src/service/CodeExecutionInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, createContext, useState } from "react";
+import { CodeExecutionInfoType } from "types";
+
+const CodeExecutionInfoContext = createContext<CodeExecutionInfoType | null>(null);
+
+const CodeExecutionInfoProvider: FC<React.ReactNode> = ({ children }) => {
+  const [value, setValue] = useState(0);
+  const [loading, setLoading] = useState(false);
+  const [inputText, setInputText] = useState("");
+  const [outputData, setOutputData] = useState({
+    output: "",
+    memory: 1,
+    cpuTime: 1,
+  });
+
+  return (
+    <CodeExecutionInfoContext.Provider
+      value={{ value, loading, inputText, outputData, setValue, setLoading, setInputText, setOutputData }}
+    >
+      {children}
+    </CodeExecutionInfoContext.Provider>
+  );
+};
+export { CodeExecutionInfoContext, CodeExecutionInfoProvider };
```

Comment:
for consistency can you please follow how the exports are done in other context [files](https://github.com/Rishabh-malhotraa/caucus/blob/main/src/service/SettingsContext.tsx)

  i.e.
``` ts
const export CodeExecutionInfoContext = createContext<CodeExecutionInfoType | null>(null);
export default CodeExecutionInfoProvider
```